app/views/projects/_form.html.erb

```diff
@@ -1,49 +1,9 @@
-<%= form_for(@project, html: { class: 'form-horizontal' }) do |f| %>
-  <% if @project.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(@project.errors.count, "error") %> prohibited this project from being saved:</h2>
+<%= bootstrap_form_for @project, class: 'form-horizontal' do |f| %>
```

Comment:
@madeindjs Could you remove the `class: 'form-horizontal'`. The `bootstrap_form_for` does not support the class option. Moreover this is currently renderering a bootstrap "vertical" style form (not an horizontal form) :smile:

Reply:
Great catch! I think we could use the `:horizontal` option, see https://github.com/bootstrap-ruby/rails-bootstrap-forms/blob/0fac4b176e753e2883b6632ab62926981cc88550/lib/bootstrap_form/helper.rb#L16.

Reply:
Corrected on https://github.com/madeindjs/tomatoes/commit/0cdd26ba4d930a72e8b640ec844f623ecceffe5f